apps/documentation/src/routes/documentation/routing/index.mdx

```diff
@@ -1,15 +1,13 @@
-import { Head } from 'tuono'
+import MetaTags from '../../../components/meta-tags'
 
-<Head>
-  <title>Tuono - Routing</title>
-  <link rel="canonical" href="https://tuono.dev/documentation/routing" />
-</Head>
+<MetaTags
+  title="Tuono - Routing"
+  canonical="https://tuono.dev/documentation/routing/intro"
+/>
 
 import Breadcrumbs, { Element } from '../../../components/breadcrumbs'
 
-<Breadcrumbs>
-  <Element label="Routing" />
-</Breadcrumbs>
+<Breadcrumbs breadcrumbs={[{ label: 'Routing' }]} />
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Possible typo: you repeated a word (ENGLISH_WORD_REPEAT_RULE)
Suggestions: `Breadcrumbs`
Rule: https://community.languagetool.org/rule/show/ENGLISH_WORD_REPEAT_RULE?lang=en-US
Category: MISC
<!-- __reviewdog__:ChBlMWQ5MDVmNDA2YWNlOTY0EgxMYW5ndWFnZVRvb2w= -->